mailarchive/mailarchive-tool/tool/pom.xml

```diff
@@ -64,6 +64,18 @@
             <groupId>javax.servlet</groupId>
             <artifactId>servlet-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
This is giving a build error in 11. All of these spring artifacts expanded out to their more specific dependencies in 10. (Like spring core)

https://confluence.sakaiproject.org/display/REL/Upgrading+a+tool+for+2.9+-+10.0+compatibility